Backend/app.py

```diff
@@ -4,182 +4,68 @@
 import api.personapi
 import api.voteapi
 import api.electionroundsapi
+import api.csvapi
 # TODO(Why is this called choiceproxyapi and no <name>api.py?)
 import api.choiceproxyapi
-
-
-from models import Person, ElectionRound, Choice, session
-from flask import Flask, request, render_template
-
+import login.auth
+from flask import Flask, request
+# testing endpoint decorators endpoint sessions
+from flask import session as fsession
+from  blueprints.personblueprint import person_Blueprint
+from  blueprints.choiceblueprint import choice_Blueprint
+from  blueprints.electionblueprint import election_Blueprint
+from blueprints.choiceproxyblueprint import choiceproxy_Blueprint
+from blueprints.voteblueprint import vote_Blueprint
+
+#Decorators import
+from login.auth import logged_in,has_role,is_present
+import configparser
+
+config = configparser.ConfigParser()
+config.read('config.ini')
+
+
+from flask import abort
+import datetime
 ''' 
 TODO List:
-- Validierung in API file auslagern
-- Statuscode ins API file
 - return json richtig parsen fuer checkin und checkout
-- readallChoices get elec info AND choices
 '''
 
 app = Flask(__name__)
-
+app.secret_key = 'some secret key to tess'

```

Comment:
was macht der key? evtl. kleiner comment im code

Reply:
ich weiß es tatsächlich nicht, der war wohl in einem der anderen Branches drin xD